test/correctness/document-correctness.py

```diff
@@ -300,40 +334,38 @@ def one_iteration(collection1, collection2, ns, seed):
 
         if indexes_first:
             for i in indexes:
-                transactional_shim.ensure_index(collection1, i)
-                transactional_shim.ensure_index(collection2, i)
-
+                # uniqueIndex = gen.global_prng.choice([True, False])
+                uniqueIndex = False
```

Comment:
Good its disabled now. We have a tracking issue #106 once it's done we can reenable this. When we do enable make sure we don't enable for 50% cases. That way unique indexes will cloud everything else. Make it something like 5-10%.